chore: v1.0.0-beta.109 (#861)
Redocly/redocly-cli: docs/changelog.md

# Redocly CLI changelog

## 1.0.0-beta.109 (2022-09-08)

> ### Features
> 
> - Added rfc7007 rule.
> - Improved error message by adding `referenced from` information.
> - Added a rule for component names validation.
> - Added a new `--format` option: `summary`.
> 
> 
> ### Fixes
> 
> - Fixed an issue with multiline strings in literal mode.
> - Fixed an issue with multiline markdown with windows-style new lines.
> - Fixed a Header object spec to require `content` or `schema`.
> - Fixed a error message for `operation-4xx-response` rule.

Reviewer: ```suggestion
### Features

- Added rfc7807 problem details rule.
- Improved error messages by adding `referenced from` information.
- Added a rule for component map names validation.
- Added a new `--format` option: `summary`.

### Fixes

- Fixed an issue with multi-line strings in literal mode.
- Fixed an issue with multi-line Markdown with Windows-style new lines.
- Fixed a Header object spec to require `content` or `schema`.
- Fixed a error message for `operation-4xx-response` rule.
```

# Redocly CLI changelog

## 1.0.0-beta.109 (2022-09-08)
### Features

- Added rfc7807 problem details rule.
- Improved error messages by adding `referenced from` information.
- Added a rule for component map names validation.

> - Added a new `--format` option: `summary`.

Reviewer: ```suggestion
- Added a new lint `--format` option: `summary`.
```

- Added notification about the new version being available.

### Fixes

- Fixed an issue with multi-line strings in literal mode.
- Fixed an issue with multi-line Markdown with Windows-style new lines.

> - Fixed a Header object spec to require `content` or `schema`.

Reviewer: ```suggestion
- Fixed the Header object type to require `content` or `schema`.
```

- Fixed a error message for `operation-4xx-response` rule.
- Fixed an issue with `paths` not being correctly handled by the `join` command.

> - Fixed the `operation-security-defined` rule to check the security on the root and in each operation.

Reviewer: ```suggestion
- Fixed and renamed the `operation-security-defined` rule to `security-defined` to check the security in the root and in each operation.
```
